src/client/pkg/require/require.go

```diff
@@ -76,6 +76,46 @@ func OneOfEquals(tb testing.TB, expected interface{}, actuals []interface{}, msg
 	}
 }
 
+// OneOfEqualsString checks one string element of a slice equals a value.
+func OneOfEqualsString(tb testing.TB, expected string, actuals []string, msgAndArgs ...interface{}) {
+	equal := false
+	for _, actual := range actuals {
+		if expected == actual {
+			equal = true
+			break
+		}
+	}
+	if !equal {
+		fatal(
+			tb,
+			msgAndArgs,
+			"Not equal : %#v (expected)\n"+
+				" one of  != %#v (actuals)", expected, actuals)
+	}
+}
```

Comment:
I guess we could change it so that it takes a `interface{}` rather than `[]interface{}` and then do runtime checks to see if it's a slice.